File: 第三周作业/词频统计.java
/**
 * Created by 晓辉 on 2016/6/9.
 */
public class 词频统计 {
    public static void main(String[] args) {
      String[] article=new String[1000];
      article= new String[]{"i","can","can"};
        String word="c";

        System.out.println(getFrequency(article,0,word));
    }
    public static int getFrequency(String[] article, int n, String word) {
        // write code here
        int count = 0;
        int a = word.length();
        for (int i = 0, len = article.length; i < len; i++) {
            for (int j = 0; j < a; j++) {
                if (article[i + j].equals(word)) {


                    count++;
                }

            }

        }
        return count;
    }
}

File: 3_week/词频统计.cpp
class Frequency {
public:
    int getFrequency(vector<string> article, int n, string word) {
        int j=0;
        for(int i=0;i<n;i++){
            if(word==article[i]){
                j++;
            }
        }
        return j;
    }
};
